submit.py

```diff
@@ -470,31 +476,33 @@ def main():
     if not args.force:
         confirm_or_die(problem, language, files, mainclass, tag)
 
-    try:
-        result = submit(submit_url,
-                        login_reply.cookies,
-                        problem,
-                        language,
-                        files,
-                        mainclass,
-                        tag,
-                        args.assignment,
-                        args.contest)
-    except requests.exceptions.RequestException as err:
-        print('Submit connection failed:', err)
-        sys.exit(1)
+    while True:
+        try:
+            result = submit(submit_url,
+                            login_reply.cookies,
+                            problem,
+                            language,
+                            files,
+                            mainclass,
+                            tag)
```

Comment:
Why did you remove `args.assignment` and `args.contest`?

Reply:
Whoops, let me bring this back!